tests/test_display.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+'''Unit tests for the display module'''
+
+# For testing purposes, clobber the rcfile
+import matplotlib
+
+matplotlib.use('Agg')  # nopep8
+
+import matplotlib.pyplot as plt
+from matplotlib.style import context as style_context
+import numpy as np
+
+# Import the hacked image comparison module
+from tests.mpl_ic import image_comparison
+
+# We'll make a decorator to handle style contexts
+from decorator import decorator
+
+import mir_eval
+import mir_eval.display
+from mir_eval.io import load_labeled_intervals
+from mir_eval.io import load_valued_intervals
+from mir_eval.io import load_labeled_events
+from mir_eval.io import load_ragged_time_series
+from mir_eval.io import load_wav
+
+
+# Test fixtures will use seaborn-muted style
+GLOBAL_STYLE = 'seaborn-muted'
```

Comment:
No, we're using matplotlib's seaborn style sheet, not seaborn itself.  It will be stable across versions.

However, when matplotlib 2 rolls out, we will have to redo the regression test fixtures.